packages/tools/replay-tool/src/helpers.ts

```diff
@@ -119,16 +125,36 @@ export async function loadContainer(
         summaryOptions: { disableSummaries: true, maxOpsSinceLastSummary: 100000 },
         gcOptions: { writeDataAtRoot: true },
     };
-    const codeLoader = new ReplayCodeLoader(new ReplayRuntimeFactory(runtimeOptions, dataStoreRegistries));
+    const codeLoader = new ReplayCodeLoader(
+        new ReplayRuntimeFactory(runtimeOptions, dataStoreRegistries, requestHandlers));
 
     // Load the Fluid document
     const loader = new Loader({
         urlResolver,
         documentServiceFactory,
         codeLoader,
-        options: {},
+        options: loaderOptions ?? {},
```

Comment:
We should pass `{ summarizeProtocolTree: true }` as the default option since it is needed for the tool to work. Callers can pass in additional options if needed.